plugins/scaffolder-backend/package.json

```diff
@@ -32,10 +32,10 @@
     "@backstage/backend-common": "^0.4.3",
     "@backstage/catalog-model": "^0.6.1",
     "@backstage/config": "^0.1.2",
-    "@backstage/integration": "^0.2.0",
+    "@backstage/integration": "^0.1.5",
```

Comment:
Should still be `0.2.0`? O_o

Same below

Reply:
Fixed, thanks!